directives/questionnaire.py

```diff
@@ -113,6 +113,14 @@ def run(self):
         }
         self.set_assistant_permissions(data)
 
+        points_set_in_arguments = False
+        if len(self.arguments) == 2 and difficulty != self.arguments[1]:
+            points_set_in_arguments = True
+
+        if points_set_in_arguments and env.aplus_quiz_total_points != points:
+            raise SphinxError('The points of the questions in the questionnaire must add up to the total points of the questionnaire!')
```

Comment:
The error message must show to the user where the problem is, i.e., the RST file and line number of the problematic questionnaire. At least this method is one way to obtain that information: `self.state_machine.get_source_and_line(self.lineno)`. You need to format the error message so that it shows the location in the beginning.

Reply:
something like this?
```
            source, line = self.state_machine.get_source_and_line(self.lineno)
            raise SphinxError("Error in " + source +" in line "+ str(line)+ ":\nThe points of the questions in the questionnaire must add up to the totalpoints of the questionnaire!")
```